### ISSUE TITLE
Getter was called with undefined data

### ISSUE THREAD

**Issue by markamoros** [pedroslopez/whatsapp-web.js#2372]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

This error is thrown when trying to block a contact using contact#block method.

This is the error trace:

```
Error: Evaluation failed: Error: Getter was called with undefined data.
    at e (https://web.whatsapp.com/:2:4828)
    at g (https://web.whatsapp.com/app.55fe470f8e81b755ec64.js:48:473631)
    at getIsMyContact (https://web.whatsapp.com/app.55fe470f8e81b755ec64.js:48:471768)
    at t.logBlockEvent (https://web.whatsapp.com/app.55fe470f8e81b755ec64.js:55:92265)
    at Object.t.blockContact (https://web.whatsapp.com/app.55fe470f8e81b755ec64.js:89:714547)
    at __puppeteer_evaluation_script__:3:45
    at ExecutionContext._evaluateInternal (/home/marc/Documents/Projects/bot/node_modules/whatsapp-web.js/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:221:19)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async ExecutionContext.evaluate (/home/marc/Documents/Projects/bot/node_modules/whatsapp-web.js/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:110:16)
    at async PrivateContact.block (/home/marc/Documents/Projects/bot/node_modules/whatsapp-web.js/src/structures/Contact.js:157:9)
```

### Expected behavior

It should block the contact

### Steps to Reproduce the Bug or Issue

Try blocking a contact

### Relevant Code

```
client = new Client({ authStrategy: new LocalAuth() });
    client.on('qr', (qr) => {
        qrcode.generate(qr, { small: true });
    });

    await client.initialize();
    const contact = await client.getContactById(CHAT_ID);
    await contact.block();
```

### Browser Type

Chromium

### WhatsApp Account Type

Standard

### Does your WhatsApp account have multidevice enabled?

No, I am not using Multi Device

### Environment

- Ubuntu 20.04.2 LTS
- "whatsapp-web.js": "^1.21.0"
- whatsapp web version: 2.2332.12
- node version: v16.13.1

**Comment by alechkos** [comment]
@markamoros 

The `block` method of `Contanct` class was fixed in #2325 that was merged but the latest changes in the library have not yet been released so in order to apply the fix on your code you should execute:
```npm
npm i github:pedroslopez/whatsapp-web.js#8ff5bceca011629ff84635d0c32951eb5119e837
```
Don't forget to close this issue :)

**Comment by alechkos** [comment]
@markamoros 

The `block` method of `Contact` class was fixed in #2325 that was merged but the latest changes in the library have not yet been released so in order to apply the fix on your code you should execute:
```npm
npm i github:pedroslopez/whatsapp-web.js#8ff5bceca011629ff84635d0c32951eb5119e837
```
Don't forget to close this issue :)
